lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/validation/JarValidator.java

```diff
@@ -87,6 +89,46 @@ public JWTClaimsSet validateRequestJwt(SignedJWT signedJWT, String clientId)
         }
     }
 
+    private void validateScope(String clientId, JWTClaimsSet claimsSet)
+            throws JarValidationException {
+        try {
+            String scope = claimsSet.getStringClaim("scope");
+            var validScopes = configService.getValidClientScopes(clientId);
+            if (validScopes.isEmpty() || !validScopes.contains(scope)) {
+                LOGGER.error(
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        "client not allowed to issue a request with this scope")
+                                .with(LOG_SCOPE.getFieldName(), scope)
+                                .with(LOG_CLIENT_ID.getFieldName(), clientId));
+                throw new JarValidationException(
+                        OAuth2Error.ACCESS_DENIED.setDescription(
+                                "Client not allowed to issue a request with this scope"));
+            }
+            LogHelper.attachClientIdToLogs(clientId);
```

Comment:
Very minor thoughts:
- Could be good to attach client id at the start so don't need to manually add them in
- Log messages could have commonly capitalised first letter